include/swift/SIL/GraphOperationInfo.h

```diff
@@ -0,0 +1,146 @@
+//===--- GraphOperationInfo.h - GraphOperationInst Parse Logic --*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines the parsing logic for a GraphOperationInst, in particular
+// decoding the mangled inst name string for the operands and attributes.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_AST_GRAPH_OPERATION_INFO_H
```

Comment:
Should this be SWIFT_SIL_GRAPH_OPERATION_INFO_H to be consistent with other files?

Reply:
Yes. Thanks for the catch!